hack/robot-e2e/e2e-setup-robot-server.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Prepare Reinstall
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    scope: dev
```

Comment:
What is this scope used for ?

Reply:
The scope is introduced in `hack/dev-up.sh` to deploy multiple clusters into the same project (ie name prefix). This was more relevant before we had unique projects per CI pipeline.